webpack.config.js

```diff
@@ -1,3 +1,11 @@
 const webpackConfig = require('./vendor/sulu/sulu/webpack.config.js');
 
-module.exports = webpackConfig;
+module.exports = (env, argv) => {
+    if (!env) {
+        env = {};
+    }
+
+    env.base_path = 'admin-build';
```

Comment:
I think that should be `build/admin`, shouldn't it?

Reply:
@danrot did fix the pr as this should only remove the cleanup from the package.json.